examples/exampleGuide.md

```diff
@@ -32,9 +31,12 @@ More [Sample templates](https://github.com/Azure/azure-quickstart-templates/tree
     parameters: https://teststorage.blob/parameters.json
 
 [Sample templates](https://github.com/Azure/azure-quickstart-templates/tree/master/managementgroup-deployments) can be found here.
+If there is a requirement to override any parameter, it can be specified with the parameter file.

```

Comment:
This is in same line as previous statement. Look at the preview. Make it a separate paragraph and me be start it with `Note:`